tests_tf/test_attacks.py

```diff
@@ -454,119 +454,6 @@ def test_attack_strength_np_batched(self):
     new_labs = np.argmax(self.sess.run(self.model.get_logits(x_adv)), axis=1)
     self.assertLess(np.mean(feed_labs == new_labs), 0.1)
 
-
-class TestBasicIterativeMethod(CommonAttackProperties):
```

Comment:
If the concern here is just that the name had "BasicIterativeMethod" it's probably better to fix it by renaming this to "TestProjectedGradientDescent".

Reply:
There were two `class TestBasicIterativeMethod`.